docs/source/using/addons.adoc

```diff
@@ -160,6 +160,9 @@ The following table shows these variables.
 
 |addon-name
 |Name of the current add-on.
+
+|user
+|Name of {project} ssh user
```

Comment:
s/Name of {project} ssh user/User used by {project} for execution of commands through ssh/ 

@minishift/minishift-dev  any better suggestions.